tools/schedule-migration/readme.md

```diff
@@ -8,13 +8,12 @@ Due to controller refactor and CRD modification, old chaos resource (1.x) won't
 
 We provide a bash script to do step 1,2 and 4. Check it out.
 
-``` bash
-./migrate.sh -h
-```
+Before running the script, you need to compile an `schedule-migrate` binary
```

Comment:
Since we provide compiled binary, I think it's better to describe how to download and use `schedule-migration.tar.gz` before compiling it yourself.

Reply:
This `readme.md` locates at the source codes, so it may need to describe how to compile the codes and generate a executable file. I will describe how to download the `schedule-migration.tar.gz` and decompress it in the update-to-2.0 document.

Reply:
👌🏽 Got it.